config/load.go

```diff
@@ -0,0 +1,89 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+)
+
+// Getter is an interface for getting
+// config file text
+type Getter interface {
```

Comment:
I believe all the interfaces should go in the files of the objects that implement the interface. I know that when an object implements an interface it is implicit but I think it is still a good idea to affirm that a struct implements a given interface

Reply:
Will switch to this.